.bazelci/presubmit.yml

```diff
@@ -6,6 +6,8 @@ tasks:
     bazel: latest
     build_targets:
     - "..."
+    build_flags:
+    - "--incompatible_remap_main_repo"
```

Comment:
you shouldn't need this

Reply:
Sadly, we do, to build stardoc targets in bazel-skylib.
See https://github.com/bazelbuild/skydoc/issues/137

I've left a comment to this effect in the BUILD file, too.